shared-helpers/src/AuthContext.ts

```diff
@@ -317,12 +319,13 @@ export const AuthProvider: FunctionComponent<React.PropsWithChildren> = ({ child
         })
         if (response) {
           dispatch(saveToken({ accessToken: response.accessToken, apiUrl, dispatch }))
-          // 12/18 - Short-term error fix to allow user accesss
-          // const profile = await userService?.userControllerProfile()
-          // if (profile) {
-          //   dispatch(saveProfile(profile))
-          //   return profile
-          // }
+          const profile = await userService?.userControllerProfile({
+            headers: { Authorization: `Bearer ${response.accessToken}` },
+          })
```

Comment:
Suggestion (non-blocking): It looks like we have this same bit of code four times in this file. A function could be created that accepts a 'token' parameter and returns the profile.

Reply:
Great point! I'll do this in a follow-up to reduce the changes here and give us the most time to test.